specifyweb/frontend/js_src/lib/components/Forms/BaseResourceView.tsx

```diff
@@ -41,6 +41,8 @@ export type ResourceViewState = {
   readonly specifyNetworkBadge: JSX.Element | undefined;
 };
 
+const tableNamesToHide = new Set(['SpAppResource', 'SpViewSetObj']);
```

Comment:
Could you make this small change? Would guard against typos and provide autocomplete

```suggestion
const tableNamesToHide = new Set<keyof Tables>(['SpAppResource', 'SpViewSetObj']);
```